### DISCUSSION TITLE
Joint TTS overfitting

### DISCUSSION THREAD

**Discussion by nellorebhanuteja** [espnet/espnet#4509] [Q&A]
Hi

I am trying to train an English Male TTS with 24 hrs data.
I ran 1000 epochs of FS2 HIFiGAN training with default conf provided
(i changed batch size due to memory issues)

The decoded outputs are not good after all.

This is how my L1 loss looks like
<img width="1410" alt="Screenshot 2022-07-12 at 10 59 11 AM" src="https://user-images.githubusercontent.com/64081481/178415560-908b5dd9-c075-4434-9839-27ecc20c12bb.png">

Can you suggest if i need to make any changes to conf?

**Comment by nellorebhanuteja** [comment]
@kan-bayashi
